target/linux/ramips/image/rt305x.mk

```diff
@@ -910,6 +910,13 @@ define Device/kn
 endef
 TARGET_DEVICES += kn
 
+define Device/kn_st
```

Comment:
Device should be named as `zyxel_keenetic-start`

Reply:
Done.